src/background/connections/middlewares/RPCCallsMiddleware.ts

```diff
@@ -10,7 +10,7 @@ export function RPCCallsMiddleware(
     const network = await networkService.getNetwork(
       context.request.params.scope
     );
-    const { method } = context.request;
+    const { method } = context.request.params.request;
```

Comment:
This was looking at the wrong method (the outer request of CAIP-27 schema - basically `provider_request`).